core/server/worker/pom.xml

```diff
@@ -77,6 +77,12 @@
       <type>test-jar</type>
       <scope>test</scope>
     </dependency>
+    <dependency>
```

Comment:
We leverage the local under file system in some integration tests. Another approach would be to include local under filesystem in common as the implementation of the under file system API.